### DISCUSSION TITLE
Can I convert a Word doc back to a Quarto markdown file?

### DISCUSSION THREAD

**Discussion by aftmnelbier11** [quarto-dev/quarto-cli#7628] [Q&A]
### Description

As far as I have read, there is not yet the possibility to collaboratively edit/work on Quarto files. I have been rendering the Quarto markdown file to a Word doc via RStudio to allow me to use Git version control. At some point though, I would have to incorporate feedback but would still like to use the version control when editing. 

**Is it possible to reverse this and convert a Word doc back to a Quarto markdown file?**

**Comment by mcanouil** [comment]
Maybe via Pandoc, but you won't get the original markdown from Quarto.
Also, even if it comes to Quarto, I would assume it is a really long term feature.

Markdown based editing seems a better solution such as using Google Collab or Obsidian.

There are several discussions over here you can search and read.
